recipes/default.rb

```diff
@@ -27,9 +27,7 @@
   action :upgrade
 end
 
-hab_package 'core/hab-studio' do
-  action :upgrade
-end
```

Comment:
Nice! Because we're calling `hab studio *` in the provider, `hab` itself will install a suitable version of `core/hab-studio` to use (including when updating `core/hab`)